e2e/cypress/integration/messaging/inline_images_open_preview_window_spec.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup({loginAfter: true}).then(({team, channel}) => {
+            // # Visit a test channel
+            cy.visit(`/${team.name}/channels/${channel.name}`);
+        });
+    });
+    it('M23345 - Inline markdown images open preview window', () => {
```

Comment:
nit: please add space in between.
I know you've followed the ticket instruction but the new format should have TM4J ID in the test description, so we can record test result in TM4J.

```suggestion
    });

    it('MM-T187 Inline markdown images open preview window', () => {
```

Reply:
Will do!